docs/index.md

```diff
@@ -97,13 +97,13 @@ package.json usage
 ## API Reference
 
 | Name        | Type      | Default                    | Description                                                                    |
-| ----------- | --------- | -------------------------- | ------------------------------------------------------------------------------ | -------------------------------------------------------------------- | ---------------------- |
-| srcDir      | `string   | string[]`                  |                                                                                | Directory to hold NW app files unless or array of file glob patterns |
-| mode        | `run      | build`                     |                                                                                | Run or build application                                             |
-| version     | `latest   | stable                     | string`                                                                        |                                                                      | NW runtime version     |
-| flavour     | `sdk      | normal`                    |                                                                                | NW runtime build flavour.                                            |
-| platform    | `linux    | osx                        | win`                                                                           |                                                                      | NW supported platforms |
-| arch        | `ia32     | x64`                       |                                                                                | NW supported architectures                                           |
+| ----------- | --------- | -------------------------- | ------------------------------------------------------------------------------ | -------------------------- | ---------------------- |
+| srcDir      | `string`  |                            | Directory to hold NW app files unless or array of file glob patterns           |
```

Comment:
- The `srcDir` description is a little confusing and the type is wrong if it allows either a string or array of strings.
- Several missing descriptions below.
- The formatting of this table is also busted and doesn't display properly on GitHub.

Reply:
Fixed the type and description for `srcDir`. Also formatted the table correctly